### ISSUE TITLE
sync process is updating all orders completion date

### ISSUE THREAD

**Issue by ziofonta** [mailchimp/mc-magento#247]
Hi, after installing and configuring the API, the sync with Mailchimp started to update a certain amout of sales orders each 5 minutes, REWRITING "completed date" to current date and time in our store. 
This is a problem since all statistics of orders need the correct completion date to work properly in our store. To stop the sync, I had to disable the cron process "mailchimp_bulksync_data" ( model mailchimp/cron::syncBatchData). I also tried to disconnect and reconnect, and we experienced the same problem of bad rewriting when syncing.
Any idea? 
Thanks

**Comment by ziofonta** [comment]
in "configuration" i tried also to change the option "Enable Two Way Sync" to "NO" hoping that no data would have changed on store, but all completion dates were rewrited anyway.

**Comment by Santiagoebizmarts** [comment]
Hello @ziofonta 
Thanks for contacting us and sorry for the delay.
This does not make sense with the latest version of the extension. Please read this [guidelines](https://github.com/mailchimp/mc-magento/wiki/Issue-reporting-guidelines) and make sure place all the information required when placing an issue.

Regards

**Comment by ziofonta** [comment]
I solved the issue by myself. Problem was caused by the fact we are using a different template than "default" for admin view. Normally, layout part of extension is expected to be installed in "app/design/adminhtml/default/default" folder. In our case, I had to copy files inside our admin theme folder to make it work properly.
Anyway, I have to notify that when the layout part is missing in correct theme folder, the behaviour is just what we experienced: all completion date are rewritten by Mailchimp.
Thanks
